tests/metadata/test_metadata_service.py

```diff
@@ -12,7 +12,8 @@
 from ossdbtoolsservice.metadata.contracts import METADATA_LIST_REQUEST, MetadataListParameters, MetadataListResponse, MetadataType, ObjectMetadata
 from ossdbtoolsservice.utils import constants
 from tests.mocks.service_provider_mock import ServiceProviderMock
-from tests.utils import MockConnection, MockCursor, MockRequestContext, MockThread
+from tests.utils import MockCursor, MockRequestContext, MockThread
+from tests.pgsmo_tests.utils import MockConnection as MockServerConnection
```

Comment:
I see that at some places we have imported this as mockConnection whereas other places as MockServerConnection. Can we keep it consistent?